tests/api/test_generic_programmable.py

```diff
@@ -0,0 +1,26 @@
+from pprint import pprint
+
+
+class TestSaiGenericProgrammable:
+    # object with no parents
+
+    def test_generic_programmable_create(self, npu):
+        commands = [
+            {
+                'name': 'generic_programmable_1',
+                'op': 'create',
+                'type': 'SAI_OBJECT_TYPE_GENERIC_PROGRAMMABLE',
+                'attributes': ['SAI_GENERIC_PROGRAMMABLE_ATTR_OBJECT_NAME', '2:10,11'],
```

Comment:
Please provide some descriptive name.

Reply:
the name is being created by the generator script which gets the name from  object SAI_OBJECT_TYPE_GENERIC_PROGRAMMABLE